docs/design/datacontracts/Loader.md

```diff
@@ -242,4 +375,11 @@ bool ILoader.IsCollectible(ModuleHandle handle)
     byte isCollectible = _target.Read<byte>(assembly + /* Assembly::IsCollectible*/);
     return isCollectible != 0;
 }
+
+bool ILoader.IsAssemblyLoaded(ModuleHandle handle)
+{
+    TargetPointer assembly = _target.ReadPointer(handle.Address + /*Module::Assembly*/);
+    uint loadLevel = _target.Read<uint>(assembly + /* Assembly::Level*/);
+    return loadLevel >= 4; // FILE_LOAD_DELIVER_EVENTS
```

Comment:
I would prefer a constant, listed out in some table at the start of the version contract. We will need to build validation for these before we can ship, and magic numbers hidden don't make this easier.